libafl/src/monitors/tui/mod.rs

```diff
@@ -27,6 +28,10 @@ use crate::{
     monitors::{ClientStats, Monitor, UserStats},
 };
 
+// Catching panics because the main thread dies
+use alloc::boxed::Box;
+use std::panic;
```

Comment:
This can be moved to the `std::{}` block above

Reply:
Ohhh yes